Client/Assets/WindowCloseHelper.js

```diff
@@ -0,0 +1,8 @@
+(function () {
+  var _close = window.close;
+  window.close = function () {
+    console.log("Yo I am going to close");
```

Comment:
Yeah I mostly want feedback. I think this patching of `window.close` is a decent approach to start until we can use the official API in iOS 9 but you know more about the JS side than I do. So I was wondering what you thought of the process in general.